data/json/items/book/traps.json

```diff
@@ -52,9 +52,8 @@
   {
     "id": "textbook_traps",
     "type": "BOOK",
-    "name": "The Compleat Trapper",
-    "name_plural": "The Compleat Trapper",
-    "//": "The spelling is intentionally archaic, as this form is commonly used in the titles of books.",
```

Comment:
Note that "[Compleat](https://www.merriam-webster.com/dictionary/compleat)" is, actually, an old English spelling of "Complete."